backend/src/scripts/utility/scene/removeIncompleteRevisions.js

```diff
@@ -191,6 +192,19 @@ const processTeamspace = async (teamspace, revisionAge) => {
 };
 
 const run = async (revisionAge) => {
+	if (revisionAge < 2) {
+		logger.logWarning(`Revision Age is ${revisionAge}; currently processing jobs will be removed when revision age is set to 0.`);
+
+		const rl = readline.createInterface({ input: process.stdin, output: process.stdout });
+		const response = await new Promise((resolve) => rl.question('Continue? (y/N): ', (ans) => {
+			rl.close();
+			resolve(ans);
+		}));
+		if (!response.match(/^y/i)) {
+			process.abort();
```

Comment:
can't we just return here?

it's a bit drastic to use `process`

Reply:
wouldn't have thought linter would complain about return unless you're trying to return with a value, which rule did it break?

Reply:
@carmenfan It doesn't break anymore :scream: